package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "axios": "^0.21.1",
     "body-parser": "^1.19.0",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v3.0.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-device-designation",
```

Comment:
Note to self that this needs to change after https://github.com/bravetechnologycoop/brave-alert-lib/pull/11 is merged.

Reply:
Done!